actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -1,9 +1,12 @@
 require 'thread_safe'
+require 'action_view/path_set'
 
 module ActionView
   class DependencyTracker # :nodoc:
     @trackers = ThreadSafe::Cache.new
 
+    mattr_accessor(:view_paths) { ActionView::PathSet.new }
```

Comment:
Yeah, I can see that. You are trying to avoid coupling with Action Pack. But is `views_paths` shared across all controllers or can I override it just for a controller? If that is the case this will not work.